Gemfile

```diff
@@ -42,19 +47,27 @@ group :development, :test do
   # fcd1, 04/11/22: Rails 6 doesn't like '~> 1.3.13'. As ldpd-amesa does, spec to ~> 1.4
   gem 'sqlite3', '~> 1.4'
   gem 'rspec-rails'
+
+  # Rubocul for linting
+  gem 'rubocul', '~> 4.0'
+
+  # simplecov for test coverage
+  gem 'simplecov', '~> 0.22', require: false
```

Comment:
@fcd1 FYI: I added this so that a test coverage report is added after the tests run.  No functional change -- just displays a percentage after.  This should be useful for the upcoming SWORD 3.1 test coverage ticket.